src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,139 @@
+.catalog {
+    --column-count: 4;
+
+    display: grid;
+    grid-template-columns: repeat(var(--column-count), 200px);
+    gap: 48px 46px;
+    margin-inline: auto;
+    padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+    .catalog {
+        --column-count: 2;
+    }
+}
+
+@media (min-width: 768px) {
+    .catalog {
+        --column-count: 3;
+    }
+}
+
+@media (min-width: 1024px) {
+    .catalog {
+        --column-count: 4;
+    }
+}
+
+.card {
+    width: 200px;
+    padding: 16px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    box-sizing: border-box;
+
+    &__product-image {
+        height: 134px;
+        margin: 16px 0 40px;
+        padding-inline: 3px;
+        background-image: url(../../images/imac.jpeg);
+        background-size: contain;
+        background-repeat: no-repeat;
+        background-position: center;
+    }
+
+    &__product-title {
+        text-align: left;
+        font-weight: 500;
+        font-size: 12px;
+        line-height: 18px;
+        margin: 0;
+    }
+
+    &__product-code {
+        margin: 4px 0 16px;
+        font-weight: 400;
+        font-size: 10px;
+        line-height: 14px;
+        text-align: left;
+        color: $secondary-color;
```

Comment:
The SCSS variable `$secondary-color` is used here but not defined. Ensure that it is defined to avoid compilation errors.